test/functional/rpc_psbt.py

```
@@ -17,7 +17,10 @@
     assert_raises_rpc_error,
     find_output
 )
-from test_framework.wallet_util import bytes_to_wif
+from test_framework.wallet_util import (
+    bytes_to_wif,
+    get_generate_key
```

Ruff (0.12.2)

Trailing comma missing

Add trailing comma

(COM812)